rotkehlchen/db/calendar.py

```diff
@@ -320,7 +326,7 @@ def query_reminder_entry(self, event_id: int) -> dict[str, list[ReminderEntry]]:
         with self.db.conn.read_ctx() as cursor:
             cursor.execute(
                 'SELECT identifier, event_id, secs_before FROM calendar_reminders '
-                'WHERE event_id=? ORDER BY secs_before ASC',
+                'WHERE event_id=? and secs_before != 0 ORDER BY secs_before ASC',
```

Comment:
why this limit? Perhaps do not allow a `secs_before` equal to 0 in the first place if you don't want this?